panel/styles.d.ts

```diff
@@ -0,0 +1,36 @@
+declare module "styles/alerts.css" {
+  export default ""
+}
+declare module "styles/booleanstatus.css" {
+  export default ""
+}
+declare module "styles/card.css" {
+  export default ""
+}
+declare module "styles/dataframe.css" {
+  export default ""
+}
+declare module "styles/debugger.css" {
+  export default ""
+}
+declare module "styles/json.css" {
+  export default ""
+}
+declare module "styles/loading.css" {
+  export default ""
+}
+declare module "styles/loadingspinner.css" {
+  export default ""
+}
+declare module "styles/markdown.css" {
+  export default ""
+}
+declare module "styles/progress.css" {
+  export default ""
+}
+declare module "styles/variables.css" {
+  export default ""
+}
+declare module "styles/widgets.css" {
+  export default ""
+}
```

Comment:
bokehjs can auto-generate this, but the extensions' compiler was not updated to support this, so I will have to do that in 3.1. However, for now this will work and when the compiler is updated, then this file can be removed

Reply:
Not needed anymore with bokeh 3.1.dev1.